bundle/jsky.app.ot/src/main/java/jsky/app/ot/gemini/gems/StrehlFeature.java

```diff
@@ -407,30 +408,41 @@ private BandsList getBandpass(GuideProbe.Type type) {
     }
 
     // Returns an array of mascot Star objects for the given target list
+    // throws an exception if a star's coordinates are unknown
     private Star[] targetListToStarList(List<SPTarget> targetList) {
         Star[] starList = new Star[3];
         int n = targetList.size();
         for (int i = 0; i < starList.length; i++) {
-            starList[i] = i < n ? targetToStar(targetList.get(i)) : null;
+            if (i < n) {
+                final SPTarget spt = targetList.get(i);
+                final Option<Star> op = targetToStar(spt);
+                if (op.isEmpty()) {
```

Comment:
I don't know, probably never. That's why I'm not too worried about throwing the exception above.